src/ValidateRecord.cpp

```diff
@@ -56,7 +56,7 @@ namespace geopm
     {
         if (hash == GEOPM_REGION_HASH_INVALID ||
             hash > UINT32_MAX) {
-            throw Exception("ValidateRecord::filter(): Region hash out of bounds",
+            throw Exception("ValidateRecord::filter(): Region hash out of bounds: " + std::to_string(hash),
```

Comment:
It would be nice to print this in hex.  You can use string_format_hex() from Helper.hpp.  Same for other hashes below.

Reply:
Done.